shell/platform/fuchsia/dart/BUILD.gn

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+assert(is_fuchsia)
+
+import("//build/fuchsia/sdk.gni")
+
+if (using_fuchsia_sdk) {
+  import("//flutter/common/config.gni")
+
+  group("dart_io_api") {
+    visibility = [ ":*" ]
+    public_deps = []
+
+    if (flutter_runtime_mode == "debug") {
+      public_deps += [ "//third_party/dart/runtime/bin:dart_io_api" ]
+    } else {
+      public_deps += [ "//third_party/dart/runtime/bin:dart_io_api_product" ]
+    }
+  }
+
+  executable("dart") {
+    output_name = "dt_rush"
```

Comment:
What does this mean?

Reply:
There were so many runners that I started naming these after cheesy Protoss [build](http://www.osirissc2guide.com/starcraft-2-dark-templar-rush-build-order.html) [orders](https://liquipedia.net/starcraft2/4_Warpgate_Rush). I should probably name these something sane.